OpenRA.Mods.Common/Projectiles/GravityBomb.cs

```diff
@@ -59,18 +59,21 @@ public GravityBomb(GravityBombInfo info, ProjectileArgs args)
 			this.info = info;
 			this.args = args;
 			pos = args.Source;
+			var actor = args.SourceActor;
 			var convertedVelocity = new WVec(info.Velocity.Y, -info.Velocity.X, info.Velocity.Z);
 			velocity = convertedVelocity.Rotate(WRot.FromFacing(args.Facing));
 			acceleration = new WVec(info.Acceleration.Y, -info.Acceleration.X, info.Acceleration.Z);
 
 			if (!string.IsNullOrEmpty(info.Image))
 			{
-				anim = new Animation(args.SourceActor.World, info.Image);
+				var ifacing = actor.TraitOrDefault<IFacing>();
+				var facing = ifacing != null ? ifacing.Facing : 0;
+				anim = new Animation(actor.World, info.Image, () => facing);
```

Comment:
Why are you querying the actor (which may crash if the actor is killed before this runs) instead of using `args.Facing` from the `ProjectileArgs`?